src/libraries/System.Private.CoreLib/src/System/Threading/Tasks/Task.cs

```diff
@@ -2637,15 +2637,44 @@ public void Wait()
         /// infinite time-out -or- timeout is greater than
         /// <see cref="int.MaxValue"/>.
         /// </exception>
-        public bool Wait(TimeSpan timeout)
+        public bool Wait(TimeSpan timeout) => Wait(timeout, default);
+
+        /// <summary>
+        /// Waits for the <see cref="Task"/> to complete execution.
+        /// </summary>
+        /// <param name="timeout">
+        /// A <see cref="TimeSpan"/> that represents the number of milliseconds to wait, or a <see
```

Comment:
I think it's wierd to write "A Timespan that represents the number of milliseconds to wait". Why not seconds or hours or microseconds?

I see thats on `SpinUntil(Func<bool> condition, TimeSpan timeout)` already. You could leave them consistent, or change both eg to "The time to wait, or -1 milliseconds to wait indefinitely."